test/unified_format.py

```diff
@@ -1087,6 +1087,25 @@ def _databaseOperation_runCommand(self, target, **kwargs):
         kwargs["command"] = ordered_command
         return target.command(**kwargs)
 
+    def _databaseOperation_runCursorCommand(self, target, **kwargs):
+        self.__raise_if_unsupported("runCursorCommand", target, Database)
+        # Ensure the first key is the command name.
+        ordered_command = SON([(kwargs.pop("command_name"), 1)])
+        ordered_command.update(kwargs["command"])
+        kwargs["command"] = ordered_command
+        if "maxTimeMS" in kwargs:
+            kwargs["max_time_ms"] = kwargs["maxTimeMS"]
+            del kwargs["maxTimeMS"]
+        return list(target.cursor_command(**kwargs))
```

Comment:
What to think about refactoring this to share the main logic with  _databaseOperation_createCommandCursor?

Reply:
Done, it now just calls `_databaseOperation_createCommandCursor` and wraps the result in a `list()`